climada/engine/test/test_cost_benefit.py

```diff
@@ -377,9 +375,9 @@ def test_npv_unaverted_no_pres_pass(self):
         cb.present_year = 2018
         cb.future_year = 2030
         risk_future = 1000
-        disc_rates = DiscRates()
-        disc_rates.years = np.arange(cb.present_year, cb.future_year + 1)
-        disc_rates.rates = np.ones(disc_rates.years.size) * 0.025
+        years=np.arange(cb.present_year, cb.future_year + 1)
+        rates=np.ones(years.size) * 0.025
+        disc_rates = years=years, rates=rates)
```

Comment:
@chahank Something is completely wrong here, and in other places in this file I see strange whitespaces. It seems like that is due to a failed attempt to do this work with search-replace patterns (?).